nimble/host/services/audio/pacs/syscfg.yml

```diff
@@ -0,0 +1,22 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+syscfg.defs:
+  BLE_SVC_AUDIO_PACS_SYSINIT_STAGE:
+    description: >
+      Sysinit stage for Published Audio Capabilities Service.
+    value: 303
```

Comment:
I think we should have MYNEWT_VAL to enable SINK and SOURCE records with a restriction that at least one have ot be choose

Reply:
Added syscfg for all characteristics, compliant with PACS Table 3.1. All optional notifications are togglable, as well as optional characteristics. At least PAC Sink or Source must be enabled